inventory/customers

```diff
@@ -0,0 +1,23 @@
+# The {cdc,rdc}_customers groups are populated by UAM's dynamic inventory.
+# Re-grouping them here allows us to add group_vars.
+
+[cloud_snitch_collectors:children]
+cdc_customers
+rdc_customers
+
+# The optout_* groups should contain environment UUIDs.
+
+[optout_global]
+32503983-21c7-41f4-bef1-fbe4893cd693
```

Comment:
is this an actual uuid?
should we be using pwsafe or something?

Reply:
Yeah, this is an actual UUID for a customer that has opted out of FDR scanning.  I *think* we are okay with UUIDs because you can't determine anything about the customer without auth'ing to additional systems (i.e. the UAM repo now, and the `rpcindex` API in the future).

I had a conversation with @chalupaul yesterday about what was/wasn't okay in an access-controlled git repo.  My understanding of the summary was:

- not okay
  - credentials (usernames, passwords, API keys/tokens, etc)
  - customer names or derivatives (including variables, group names, domain names, etc).  basically, anything that if exposed would suggest a relationship between Rackspace and the customer.
  - customer IP addresses
  - customer CORE account numbers
- okay
  - internal (but not customer-related) IPs
  - internal (but not customer-related) host/domain names
  - pretty much anything else that isn't secret/sensitive